Makefile

```diff
@@ -218,7 +218,11 @@ install-doc-symlink:
 	for FILE in $(shell cd doc ; ls) ; do \
 		ln -fs "$(PWD)/doc/$$FILE" "${DESTDIR}${DOCDIR}" ; done
 
-install love: install-doc install-man install-www install-pkgconfig
+install-r2sdb:
+	mkdir -p "${DESTDIR}${BINDIR}"
+	cp -f shlr/sdb/sdb "${DESTDIR}${BINDIR}/r2sdb"
```

Comment:
use INSTALL_PROGRAM instead of cp

Also, add support for symlink install of r2sdb

Reply:
also use $(SHLR)/sdb instead of shlr/sdb